src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx

```diff
@@ -25,6 +25,7 @@ interface Props {
   measureGroups: Group[];
   groupCoverageResult: GroupCoverageResult;
   cqlDefinitionCallstack;
+  includeSDE;
```

Comment:
Can we type the properties, especially when declaring in an interface ?

Reply:
@RohitKandimalla added boolean type.